### ISSUE TITLE
Helm Chart for Deployments

### ISSUE THREAD

**Issue by Joeskyyy** [aws/amazon-eks-pod-identity-webhook#4]
**What would you like to be added**: Helm chart for standardised deployment

**Why is this needed**: Helm is widely used in the k8s community for deploying resources across clusters. This would allow for teams to bake existing workflows in things like terraform, ansible, etc. more easily than the current Makefile, as well as allow more customisation where needed for teams that may have different access control needs or configuration needs wrt some of the resources created for this.

---

Looking at the Makefile, I don't think this is an insurmountable task at all :D Looks like the only iffy part might be the cert generation?

If I can find the spare cycles, I might just pick this one up myself.

**Comment by micahhausler** [comment]
Hey thanks, I think a Helm chart would be great, and I'd be happy to be a co-maintainer on one in https://github.com/helm/charts/. I think we'd want to keep it in the helm/charts repository so we can get all the CI/release goodness that is automatically setup there

**Comment by Joeskyyy** [comment]
@micahhausler agreed on putting it in the helm repo, figured I'd file the FR here for tracking for the EKS side of the house. (:

**Comment by max-rocket-internet** [comment]
> I think the name is great

🎉

> you should probably poke some org admin who can invite the listed OWNERS. I'll have @jqmichael and @nckturner comment on the chart PR with their owner/org request status

I'll try but I don't see much movement in that area these days. It seems it's always the same people (and bottleneck) for reviewing chart PRs.

**Comment by hlascelles** [comment]
NB, for anyone arriving here: @max-rocket-internet has closed the [initial PR](https://github.com/helm/charts/pull/17099) opened in the main helm repo in favour of one in the aws repo: https://github.com/aws/eks-charts/pull/28 (also opened by him). Upvote it there (bearing in mind it is [stalled on](https://github.com/aws/amazon-eks-pod-identity-webhook/issues/5) an a docker image build).

**Comment by danopia** [comment]
For continuation, there's a new PR in the aws repo now that a docker image has been published: https://github.com/aws/eks-charts/pull/286

**Comment by jkroepke** [comment]
In meantime, I provide a helm chart which optionally support the cert-manager or static generated tls certificates.

https://github.com/jkroepke/helm-charts/tree/main/charts/amazon-eks-pod-identity-webhook

**Comment by LockedThread** [comment]
Any updates on this?
